nautobot/dcim/urls.py

```diff
@@ -45,6 +45,7 @@
 router.register("module-bays", views.ModuleBayUIViewSet)
 router.register("module-bay-templates", views.ModuleBayTemplateUIViewSet)
 router.register("module-types", views.ModuleTypeUIViewSet)
+router.register("virtual-device-contexts", views.VirtualDeviceContextUIViewSet)
```

Comment:
Looks like the API urls use `vdcs` - let's pick one (I prefer `virtual-device-contexts`) and be consistent.